php/Terminus/Models/Site.php

```diff
@@ -332,6 +333,19 @@ public function fetch(array $options = array()) {
   }
 
   /**
+   * Re-fetches site attributes from the API
+   *
+   * @return void
+   */
+  public function fetchAttributes() {
+    $response = $this->request->simpleRequest(
+      sprintf('sites/%s/settings', $this->get('id'))
+    );
+    $this->attributes = $response['data'];
+    $this->collection->sites_cache->update((array)$response['data']);
```

Comment:
This is the only data we should be storing in the cache:
https://github.com/pantheon-systems/terminus/blob/master/php/Terminus/Caches/SitesCache.php#L58-L69

I would rather cache even less than that, but it's the minimum needed to generate the table of _all sites_. Individual site commands should always use the API to fetch and freshen data and not rely on the cache other than the initial sitename->id lookup.